docs/source/conf.py

```diff
@@ -3023,6 +3023,7 @@
     "GroupNorm",
     "LayerNorm",
     "LocalResponseNorm",
+    "RMSNorm",
```

Comment:
Given that you updated the .rst files, I expect this is not needed?
Also adding new entry here is not allowed!

Reply:
If I do not add this here it complains about "forward", "__repr__" and "reset_parameters" in `torch.nn.normalizations` not being documented, but none of the other nn modules document this

Reply:
`__repr__` sounds suprising?
reset_parameters should most likely be documented? forward would make sense as well.

In any case, we have a pretty strict rule that nothing should be added here I'm afraid.

Reply:
Sorry it was `extra_repr` not `__repr__`

This is the output of coverage.txt
```
torch.nn.modules.normalization
------------------------------
Classes:
 * RMSNorm -- missing methods:

   - extra_repr
   - forward
   - reset_parameters
   ```
   
   If I uncomment any nn.Module in docs/source/conf.py they will have similar errors.
   
   
   Does this mean you want us to document `torch.nn.modules.normalization`? Removing this line will make the error go away
   
https://github.com/pytorch/pytorch/blob/3db64c19554d644d09d2a16931bae7ebe96087fc/docs/source/nn.rst?plain=1#L556